src/entries/popup/hooks/useHomeShortcuts.ts

```diff
@@ -104,7 +105,9 @@ export function useHomeShortcuts() {
       const activeAppWalletSwitcher =
         appConnectionSwitchWalletsPromptIsActive();
       const inputIsFocused = getInputIsFocused();
+      const isModal = getActiveModal();
       if (inputIsFocused) return;
+      if (isModal) return;
```

Comment:
Poorly named I guess, but these are actually the global shortcuts here. We'd expect `w` to open the wallet switcher from any of the layer 1 drill-downs

Reply:
ok cool cool. I can see what I can do in that case.